spec/models/item_spec.rb

```diff
@@ -1,12 +1,21 @@
 describe Item do
   before(:each) do
-    FactoryGirl.create(:item)
+    @item = create(:item_with_3_reviews)
   end
```

Comment:
Instead of using instance variables in a before block, do the following:

```
describe Item do
  let!(:item) { create(:item_with_3_reviews) }

   describe "#price" do
    it ...
end
```